src/main/connector/system/Linux/Linux.yaml

```diff
@@ -0,0 +1,156 @@
+extends:
+- ../System/System
+metrics:
+  system.service.loaded:
+    description: Service Loaded State
+    type:
+      stateSet:
+        - loaded
+        - not-found
+  system.service.active:
+    description: Service Active State
+    type:
+      stateSet:
+        - active
+        - inactive
+  system.service.sub:
+    description: Service Sub State
+    type:
+      stateSet:
+        - running
+        - dead
+        - exited
+connector:
+  displayName: "LinuxOS"
+  platform: "Any platform with LinuxOS"
+  reliesOn: "Linux OsCommands"
+  information: "Gives OS specific information and metrics"
+  detection:
+    connectionTypes:
+    - remote
+    - local
+    appliesTo:
+      - "linux"

```

Comment:
Nitpicking: for an array, you don't need to add extra indentation spaces before the dash (`-`), just like you did for `connectionTypes` on lines 29, 30, and 31. 

This avoid having too much indentation.

> **Note**
> The `- ` is considered as the indentation itself, that's why it's not required in YAML. See [YAML spec](https://yaml.org/spec/1.2.2/#21-collections).